security/index.adoc

```diff
@@ -0,0 +1,90 @@
+* xref:../security/certificate_types_descriptions/olm-certificates.adoc#cert-types-olm-certificates[OLM certificates]
+* xref:../security/certificate_types_descriptions/user-provided-certificates-for-default-ingress.adoc#cert-types-user-provided-certificates-for-default-ingress[User-provided certificates for default ingress]
+* xref:../security/certificate_types_descriptions/ingress-certificates.adoc#cert-types-ingress-certificates[Ingress certificates]
+* xref:../security/certificate_types_descriptions/monitoring-and-cluster-logging-operator-component-certificates.adoc#cert-types-monitoring-and-cluster-logging-operator-component-certificates[Monitoring and cluster logging Operator component certificates]
+* xref:../security/certificate_types_descriptions/control-plane-certificates.adoc#cert-types-control-plane-certificates[Control plane certificates]
+
+[discrete]
+=== Encrypting data
+
+You can xref:../security/encrypting-etcd.adoc#encrypting-etcd[enable etcd encryption] for your cluster to provide an additional layer of data security. For example, it can help protect the loss of sensitive data if an etcd backup is exposed to the incorrect parties.
+
+[discrete]
+=== Vulnerability scanning
+
+Administrators can use the Container Security Operator (CSO) to run xref:../security/pod-vulnerability-scan.adoc#pod-vulnerability-scan[vulnerability scans] and review information about detected vulnerabilities.
+
+[id="compliance-overview"]
+== Compliance overview
+
+For many {product-title} customers, regulatory readiness, or compliance, on some level is required before any systems can be put into production. That regulatory readiness can be imposed by national standards, industry standards or the organization's corporate governance framework.
+
+[discrete]
+=== Compliance checking
+
+Administrators can use the xref:../security/compliance_operator/compliance-operator-understanding.adoc#understanding-compliance-operator[Compliance Operator] to run compliance scans and recommend remediations for any issues found. The xref:../security/compliance_operator/oc-compliance-plug-in-using.adoc#using-oc-compliance-plug-in[`oc-compliance` plug-in] is an OpenShift CLI (`oc`) plug-in that provides a set of utilities to easily interact with the Compliance Operator.
+
+// TODO: Renamed "Compliance Operator" to just "Compliance" and moved the oc-compliance plug-in assembly under it. Check if okay, and if so, does it need a redirect?
+
+[discrete]
+=== File integrity checking
+
+Administrators can use the xref:../security/file_integrity_operator/file-integrity-operator-understanding.adoc#understanding-file-integrity-operator[File Integrity Operator] to continually run file integrity checks on cluster nodes and provide a log of files that have been modified.
+
+[id="additional-resources_security-compliance-overview"]
+== Additional resources
+
+* xref:../authentication/understanding-authentication.adoc#understanding-authentication[Understanding authentication]
```

Comment:
For Additional resources, the first link was referenced earlier in the topic. Do we want to point to the same reference material twice?

Reply:
@TanyaShearonRH Can you specify which link you're talking about? If it's this "Understanding authentication" link, then I think this is the only place we mention it.